app/helpers/_flashes_helper.rb

```diff
@@ -0,0 +1,22 @@
+module FlashesHelper
+
+  ACCEPTED_FLASH_HTML_CLASSES = {
+    success: 'alert alert-success',
+    error: 'alert alert-danger'
```

Comment:
We decided at the scrum from 1st Jan that we will only use these 2. (green and red)
Not sure what to say about dismissible alerts. They look cool.